spec/controllers/api/v1/invoice_items_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::InvoiceItemsController do
+  fixtures :invoice_items
+  describe "GET index" do
+    it "can get all invoice_items" do
+      get :index
+
+      assert_response :success
+
+      parsed_invoice_items = JSON.parse(response.body)
+
+      expect(parsed_invoice_items.count).to eq 4
+    end
+  end
+
+  describe "GET show" do
+    it "can get a single instance of invoice_item" do
+    invoice_item = invoice_items(:one)
+      get :show, id: invoice_item.id
```

Comment:
this causes a deprecation warning, we should use this syntax: 

* params: { id: invoice_item.id }